### DISCUSSION TITLE
New install on Win 11 : need help

### DISCUSSION THREAD

**Discussion by Stef198** [chuot/rdio-scanner#456] [Q&A]
Hi All, 

I DL the windows zip and copied the 2 files in the repo on my Win 11.

I have copied paste the link url  http://user01-pc:3000 and open the web page..... No talkgroup.

How do I import the talkgroup from RReference in the webpage http://user01-pc:3000/admin  since I have a premium account?   The web page doesn't want to  read the csv file.   I would like to have the same database like the android app have.  Fire for all the Quebec (canada) region.

Anyone was able to setup properly here?

S

**Comment by Stef198** [comment]
UPDATE:   I was able to create a System and import Freq TG... but I don,t hear anything.   What do I need to do more or install?

Do I need to connect my own Scanner?

**Comment by rdmcvicar** [comment]
Assuming I understand the questions, the short answer is: Yes - you will need your scanner or some kind of actual radio.

What it sounds like you've done so far is create a place to bring and organize audio and metadata (such as the system talkgroups) so you can listen to it live and/or play back older traffic.  rdio-scanner is terrific for this.  But...

That audio in almost all cases needs to come from some sort of radio.  Your own scanner might work if combined with ProScan or some sort of software that'll turn received traffic into files which rdio-scanner can ingest.  Another, if not more popular, option would be one or more Software Defined Radios combined with some other piece of software (trunk-recorder, SDRtrunk, etc.) that'll electronically manipulate those SDRs to catch what you're listening for and send it to the rdio-scanner instance you've created.

Said another way, there won't be anything to listen to by running the rdio-scanner software (even if pre-populated with TGs/names from RadioReference) by itself.

Hope that helps...

**Comment by Stef198** [comment]
Thanks, that does help.

 I have a BCD325P2 but have no idea how to setup the talkgroup properly in the scanner and Rdio to either ear and push.

The info PDF if not to user friendly.   I will try mony things if no one here have a good hand to help me out.

**Reply by AvalancheAddict** [reply]
Most people that use Rdio Scanner use it in conjunction with SDR's (software defined radios) as a means to take the audio files captured by their SDR's and have Rdio Scanner essentially repackage them so that they play more like a scanner.  While you can ingest files from your Uniden, it probably isn't worth the trouble if that's all you are using for a radio.  Where Rdio Scanner can also come in handy when it comes to scanners is if you are using a combination of SDR's along with scanners.  When doing that there are creative ways to blend both the SDR recordings and recordings from the scanners all into one place (kind of making one big scanner with everything).

I would say Rdio Scanner isn't really going to do much for you if you are just using your BCD325P2, but if you enter the world of SDR's, then you may want to revisit it.  If you do get into and understand SDR's, the documentation for Rdio Scanner will make a lot more sense,

Good Luck.
